doc/modules/ROOT/pages/cops_layout.adoc

```diff
@@ -3165,9 +3165,9 @@ are indented one step.
 In Ruby 2.3 or newer, squiggly heredocs (`<<~`) should be used. If you
 use the older rubies, you should introduce some library to your project
 (e.g. ActiveSupport, Powerpack or Unindent).
-Note: When ``Layout/LineLength``'s `AllowHeredoc` is false (not default),
+Note: When `Layout/LineLength`'s `AllowHeredoc` is false (not default),
```

Comment:
I think in such cases you need double backticks in AsciiDoc or it won't render properly.